### PR TITLE
I-ALiRT: Create project

### PR DESCRIPTION
**Pull request by laspsandoval**
# Change Summary

## Overview
Creating a project for the I-ALiRT data access repo

## New Files
- .github/workflows
   - release.yml
   - test.yml
- .pre-commit-config.yaml
- codecov.yml
- poetry.lock
- pyproject.toml
- .gitignore
- ialirt_data_access/__init__.py
- tests/__init__.py

## Testing
(venv) MacL4581:ialirt-data-access lasa6858$ curl --data-binary @codecov.yml https://codecov.io/validate
Valid!

- Checked that pre-commit found errors sufficiently
- Verified that environment was properly installed
- Github Workflow for tests working properly (as shown below)

### CODE REVIEW

**Review by maxinelasp — APPROVED** [review]
LGTM! Are you planning on depending on imap-data-access in this, or will they be separate? There is some tooling in imap-data-access that would probably be useful to reduce duplicate work here.

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    - id: mixed-line-ending
+    - id: trailing-whitespace
+    - id: no-commit-to-branch
+      args: [--branch, main, --branch, dev]
+  - repo: https://github.com/astral-sh/ruff-pre-commit
+    rev: 'v0.3.4'
+    hooks:
+    - id: ruff
+      args: [--fix]
+    - id: ruff-format
```
Comment (lacoak21) [inline .pre-commit-config.yaml:22]: Might be worth it to add mypy, or poetry-lock and poetry-check. I noticed the imap_processing pre-commit has these.

**Review by lacoak21 — APPROVED** [review]
Looks good! I only had one comment about the pre commit checks.

File: pyproject.toml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+name = "ialirt_data_access"
+version = "0.1.0"
+description = "I-ALiRT Data Access"
+authors = ["IMAP SDC Developers <imap-sdc@lists.lasp.colorado.edu>"]
+readme = "README.md"
+license = "MIT"
+keywords = ["IMAP", "SDC", "SOC", "SDS", "Science Operations"]
+classifiers = [
+    "Development Status :: 3 - Alpha",
+    "License :: OSI Approved :: MIT License",
```
[the remark this answers is not in the document]
Comment (laspsandoval) [reply pyproject.toml:15]: Thanks for the heads up!

**Pull request merged**